src/components/v5/common/ActionSidebar/partials/Motions/Motions.tsx

```diff
@@ -79,6 +82,17 @@ const Motions: FC<MotionsProps> = ({ transactionId }) => {
     stopPollingForAction,
   ]);
 
+  useEffect(() => {
+    // previousIsFinalized can be undefined when opening an already closed action
+    if (motionData?.isFinalized && previousIsFinalized === false) {
+      /**
+       * We need to remove all getDomainBalance queries once a payment or funding has been successfully completed
```

Comment:
We currently do this for all finalized motions right? do we need to check for the action type and maybe add it to  the `useFinalizeSuccessCallback` helper hook?
![image](https://github.com/user-attachments/assets/ee7218a6-596f-4c86-bf41-d4d7ec583495)